example/App.js

```diff
@@ -154,7 +154,7 @@ const styles = StyleSheet.create({
   },
   value: {
     backgroundColor: 'green',
-    color: 'white',
+    color: 'black',
```

Comment:
Must be changed to black because backgroundColor is not yet supported on this version of React Native Windows.